src/main/scala/scalismo/statisticalmodel/RealignExtendedBasis.scala

```diff
@@ -0,0 +1,71 @@
+package scalismo.statisticalmodel
+
+import breeze.linalg.DenseMatrix
+import scalismo.common.DiscreteDomain
+import scalismo.geometry.*
+
+/**
+ * types the whole discrete low rank gp to make sure that it is applied to the appropriate models. The value type could
+ * be left out if the user knows what to do.
+ */
+trait RealignExtendedBasis[D: NDSpace, Value]:
```

Comment:
Do we need the NDSpace context bound here? If not, it should be omitted.

Reply:
you are right. good catch